docs/modules/ROOT/pages/documentation/payara-server/public-api/openid-connect-support.adoc

```diff
@@ -396,6 +403,83 @@ With the `logout` parameter of the `OpenIdAuthenticationDefinition` you can defi
+
+When fields, which are lists (e.g. `scopesSupported`, `responseTypesSupported`), are loaded from MicroProfile Config, they are separated by a comma, following https://download.eclipse.org/microprofile/microprofile-config-2.0/microprofile-config-spec-2.0.html#_array_converters[MicroProfile Config Array conventions].
+
+.Provider Metadata Options
+|===
+| Option | MP Config property | Required | Description
+
+| `issuer`
+| `payara.security.openid.provider.issuer`
+| true
+| The base address of OpenId Connect Provider.
+
+| `authorizationEndpoint`
+| `payara.security.openid.provider.authorizationEndpoint`
+| true
+| The URL for the OAuth2 provider to provide authentication.
+
+| `tokenEndpoint`
+| `payara.security.openid.provider.tokenEndpoint`
+| true
+| The URL for the OAuth2 provider to give the authorization token.
+
+| `userinfoEndpoint`
+| `payara.security.openid.provider.userinfoEndpoint`
+| true
+| An OAuth 2.0 Protected Resource that returns Claims about the authenticated End-User.
+
+| `endSessionEndpoint`
+| `payara.security.openid.provider.endSessionEndpoint`
+| false
+| OP endpoint to notify that the End-User has logged out of the site and might want to log out of the OP as well.
+
+| `jwksURI`
+| `payara.security.openid.provider.jwksURI`
+| true
+| An OpenId Connect Provider's JSON Web Key Set document.
+
+| `scopesSupported`
+| `payara.security.openid.provider.scopesSupported`
+| recommended
+| List of the OAuth 2.0 scope values that this server supports, e.g. `openid`.
+
+| `responseTypesSupported`
+| `payara.security.openid.provider.responseTypeSupported`
+| true
+| List of the OAuth 2.0 response_type values that this OP supports, e.g. `code`, `id_token`, `token id_token`.
+
+| `subjectTypesSupported`
+| `payara.security.openid.provider.subjectTypesSupported`
+| true
+| List of the Subject Identifier types that this OP supports. Valid types include `pairwise` and `public`.
+
+| `idTokenSigningAlgValuesSupported`
+| `payara.security.openid.provider.idTokenSigningAlgorithmsSupported`
+| true
+| List of the JWS signing algorithms (alg values) supported by the OP for the ID Token to encode the Claims in a JWT, e.g. `RS256`.
+
+| `idTokenEncryptionAlgValuesSupported`
+| `payara.security.openid.provider.idTokenEncryptionAlgValuesSupported`
+| false
+| List of the JWE encryption algorithms (`alg` values) supported by the OP for the ID Token to encode the Claims in a JWT.
+
+| `idTokenEncryptionEncValuesSupported`
+| `payara.security.openid.provider.idTokenEncryptionEncValuesSupported`
+| false
+| List of the JWE encryption algorithms (`enc` values) supported by the OP for the ID Token to encode the Claims in a JWT.
+
+| `claimsSupported`
+| `payara.security.openid.provider.claimsSupported`
+| recommended
+| List of the Claim Names of the Claims that the OpenID Provider MAY be able to supply values for. Note that for privacy or other reasons, this might not be an exhaustive list.
```

Comment:
```suggestion
| List of the Claim Names of the Claims that the OIDC Provider MAY be able to supply values for. Note that for privacy or other reasons, this might not be an exhaustive list.
```